src/bundle/Resources/views/themes/admin/content_type/tab/view.html.twig

```diff
@@ -64,7 +64,7 @@
         </tr>
         <tr class="ez-table__row">
             <td class="ez-table__cell">{{ "content_type.container"|trans|desc("Container") }}</td>
-            <td class="ez-table__cell">{{ 'yes_no'|transchoice(content_type.isContainer)|desc("{0}No|{1}Yes") }}</td>
+            <td class="ez-table__cell">{{ 'yes_no'|transchoice(content_type.isContainer ? 1 : 0)|desc("{0}No|{1}Yes") }}</td>
```

Comment:
This filter is deprecated, can we use `trans` here [as suggested in the doc](https://symfony.com/doc/current/reference/twig_reference.html#transchoice)?

Reply:
Ok, I'm ready to trade approve for a follow-up JIRA issue number :P

Reply:
https://jira.ez.no/browse/EZP-30676